03.bowling/bowling.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+points = ARGV[0].split(',')
```

Comment:
```suggestion
points = ARGV[0].gsub('X', '10').split(',')
```

この時点でgsubしちゃってもいいかもしれません

Reply:
map!の修正と合わせて修正しましたー